weather/www/index.html

```diff
@@ -0,0 +1,219 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Weathereather</title>
+    <meta name="viewport" content="initial-scale=1, maximum-scale=1, user-scalable=no, width=device-width">
+
+    <link href="lib/ionic/css/ionic.css" rel="stylesheet">
+    <link href="css/style.css" rel="stylesheet">
+    <script src="lib/ionic/js/ionic.bundle.js"></script>
+
+    <!-- Needed for Cordova/PhoneGap (will be a 404 during development) -->
+    <script src="js/app.js"></script>
+    <script src="js/services.js"></script>
+    <script src="cordova.js"></script>
+  </head>
+
+  <body ng-app="weather">
+      <ion-header-bar class="bar-dark">
+        <!--  <button class="button butto-icon" ng-click="toggleProjects()">
+              <i class="icon ion-navicon"></i>
+            </button>-->
+        <h1 class="title">
+          <emi>Wenham, MA
+          </emi>
+        </h1>
+      </ion-header-bar>
+
+      <ion-content class="background_color" ng-controller="HomeCtrl" scroll="true">
+        <ion-slide-box show-pager="false">
+          <ion-slide>
+            <div class="row">
+              <div class="col-100">
+                <h2 sytle="font-size = 200%">
+                  {{currentDate}}
+                </h2>
+                <h2>
+                  {{summary}}
```

Comment:
*Read the comment on your javascript first*
As an example, once you set ``$scope.weather = response.data `` in your javascript.
to access the summary here you would do `` {{ weather.currently.summary }}`` because summary is a property of the Json object.
Let me know if that doesn't make sense :)